src/librbd/AioImageRequest.h

```diff
@@ -59,11 +60,17 @@ class AioImageRequest {
   AioCompletion *m_aio_comp;
   Extents m_image_extents;
   bool m_bypass_image_cache = false;
+  const ZTracer::Endpoint endp;
```

Comment:
`Endpoint` stores the name of the subsystem and an optional address/port (only the Messenger's `Endpoint` fills those in). all traces in that subsystem can share the same `Endpoint`, so this probably belongs in some higher level object instead - maybe the image context? a static would work too. we should also probably name it "rbd" or "librbd", rather than "AioImageRequest"